recipe/meta.yaml

```diff
@@ -2,43 +2,40 @@
 {% set ver = version.replace(".", "_") %}
 
 package:
-    name: esmpy
-    version: {{ version }}
+  name: esmpy
+  version: {{ version }}
 
 source:
-#    git_url: git://git.code.sf.net/p/esmf/esmf
-#    git_tag: ESMF_7_1_0r
-    url: http://www.earthsystemmodeling.org/esmf_releases/public/ESMF_7_1_0r/esmf_7_1_0r_src.tar.gz
-    sha256: ae9a5edb8d40ae97a35cbd4bd00b77061f995c77c43d36334dbb95c18b00a889
+  url: http://www.earthsystemmodeling.org/esmf_releases/public/ESMF_{{ ver }}/esmf_{{ ver }}_src.tar.gz
+  sha256: ae9a5edb8d40ae97a35cbd4bd00b77061f995c77c43d36334dbb95c18b00a889
 
 
 build:
-    number: 0
-    skip: True  # [win]
+  number: 1
+  skip: True  # [win]
 
 requirements:
-    build:
-        - python
-        - numpy
-        - esmf {{ version }}
-
-        - nose  # Needed for testing
```

Comment:
Removed nose as a build dependency and it is not even a test dependency b/c we are not running all the tests now.